GitUI/Themes/README.md

```diff
@@ -0,0 +1,23 @@
+﻿GitExtensions stores themes in *.css files.
```

Comment:
The readme is a great idea, but it needs some work.

It would be great to add sections, may be something like the following:

```md
# Overview

# Themes locations

### Installed app

* Official themes

...

* Custom themes

...

### Portable app

# How to create custom theme
```

It needs to discuss that there are two distinct modes - installed and portable, and each mode handles themes differently. I.e. the portable mode supports only local themes (those placed in .\Themes) folder, whereas the installed app supports official themes and user-defined themes. The official themes are located in the app installation folder (typically `%ProgramFiles(x86)%\GitExtensions\Themes`), and these mustn't be changed.

Reply:
ok